client/app/views/steps/infos.coffee

```diff
@@ -1,10 +1,139 @@
 StepView = require '../step'
+_ = require 'underscore'
+
 
 module.exports = class InfosView extends StepView
     template: require '../templates/view_steps_infos'
 
+    ui:
+        next: '.controls .next'
+
     events:
-        'click button': 'onSubmit'
+        'click @ui.next': 'onSubmit'
+
+    fields: [
+        'public_name',
+        'email',
+        'timezone'
+    ]
+
+    initialize: (args...) ->
+        super args...
+
+        # Method used to update submit state when text is typed into inputs.
+        # Throttled to avoid too much calls.
+        @updateSubmitState = _.throttle( =>
+            if @formIsFilled()
```

Comment:
Ok, I agree with your points here. That was just a suggestion but I think we are good. Thanks 👍